lib/utils/getMoneyMultiplier.js

```diff
@@ -0,0 +1,7 @@
+export function getMoneyMultiplier(stripes, recordCurrency) {
```

Comment:
can it be stripes independent?

Reply:
yes, but what the reason? It's location is `stripes-acq-components`, so I find it logical to depend on stripes object. However, it uses only `locale` and `currency` from there. So also it's handy to pass only one argument instead of 2

Reply:
updated. used only currency, no `stripes`